climada/hazard/test/test_trop_cyclone.py

```diff
@@ -46,59 +46,64 @@ class TestReader(unittest.TestCase):
 
     def test_set_one_pass(self):
         """Test _tc_from_track function."""
+        intensity_idx = [0, 1, 2,  3,  80, 100, 120, 200, 220, 250, 260, 295]
+        intensity_values = {
+            "geosphere": [25.76590964, 27.08333002, 28.46008202, 25.70445069, 31.45216632,
+                          36.45564037, 21.22679800, 28.22022122, 32.92315537, 31.60115745, 0,
+                          40.62433745],
+            "equirect": [25.76575952, 27.08314419, 28.45984342, 25.70460801, 31.45171899,
+                         36.45003013, 21.22671287, 28.22016474, 32.92054837, 31.59980326, 0,
+                         40.62068325]
+        }
```

Comment:
Compare these values to get an idea of the effective loss in accuracy for the two distance metrics.